wp-theme-2018/sidebar.php

```diff
@@ -11,20 +11,74 @@
 
 global $EPFL_MENU_LOCATION;
 
+function get_rendered_sider_item($crumb_item, $currentPage, $children) {
+    $title = $crumb_item['title'] ?? '';
+    $url = $crumb_item['url'] ?? '';
 
-// to display correctly the menu on level 1 pages, on missing menu, or for the blog home,
-// we need to add '.current-menu-parent' to the wrapper
-$classes = '';
+    if ($crumb_item['title'] == $currentPage->title) {
+        $subMenus = '<ul class=\"sub-menu\">';
```

Comment:
Please stick to the same naming convention for your variables. I suggest `$submenus` here.